.gitignore

```diff
@@ -41,6 +41,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.idea/
```

Comment:
This belongs in your [global `.gitignore`](https://ddg.gg/global%20gitignore), not here.